code-scanning/black-duck-security-scan-ci.yml

```diff
@@ -0,0 +1,54 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
Hi @sadmananik it looks like the linter is failing on this document. Could you remove additional white spaces?

Reply:
Should be fixed.